cli_tools/diagnostics/rdp_status.ps1

```diff
@@ -2,6 +2,14 @@ $RDPTCPpath = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\Winstation
 $RDPServerPath = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server'
 
 $items= @(
+  @{
+    description="Verify that the Ethernet adapter is active, expect [Enabled]:";
+    cmd="netsh interface show interface"
+  },
+  @{
```

Comment:
Static IPs are something that we support as they're needed for specific use cases such as domain controllers. 

So obtaining the current IP is great, but the correct evaluation is that the IP matches the instance information, not that it's set to DHCP.